streams/src/main/java/org/apache/kafka/streams/state/RocksDBConfigSetter.java

```diff
@@ -29,12 +32,31 @@
  */
 public interface RocksDBConfigSetter {
 
+    Logger LOG = LoggerFactory.getLogger(RocksDBConfigSetter.class);
+
     /**
      * Set the rocks db options for the provided storeName.
      * 
      * @param storeName     the name of the store being configured
-     * @param options       the Rocks DB options
+     * @param options       the RocksDB options
      * @param configs       the configuration supplied to {@link org.apache.kafka.streams.StreamsConfig}
      */
     void setConfig(final String storeName, final Options options, final Map<String, Object> configs);
+
+    /**
+     * Close any user-constructed objects that inherit from {@link org.rocksdb.RocksObject RocksObject}.
+     * <p>
+     * Any object created with @{code new} in {@link RocksDBConfigSetter#setConfig setConfig()} and that inherits
+     * from {@link org.rocksdb.RocksObject RocksObject} should have {@link org.rocksdb.RocksObject#close() close()}
```

Comment:
Sorry for bringing this up again. I generated the JavaDocs with `./gradlew aggregatedJavadoc` and there are no links generated for `RocksObject` and any other class outside Kafka. What we get is a non-fully qualified name in `@code` font, i.e., `RocksObject`. I would at least ensure that the fully-qualified name is displayed. See also the other occurrences of links to classes outside Kafka.

Reply:
Ah, thanks for actually checking here...I meant to but "it worked in my IDE"... 

Agreed that a fully qualified name here is important if we can't properly link though, I'll change these up yet again